firmware/boron-ins-fsm/src/stateMachine.cpp

```diff
@@ -241,21 +267,23 @@ void state3_stillness() {
         saveStateChangeOrAlert(3, 2);
         stateHandler = state0_idle;
     }
+    else if (millis() - state3_stillness_timer >= *max_stillness_time) {
+        Log.warn("stillness alert, remaining in state3 after publish");
+        publishStateTransition(3, 3, checkDoor.doorStatus, checkINS.iAverage);
+        saveStateChangeOrAlert(3, 5);
+        Log.error("Stillness Alert!!");
+        Particle.publish("Stillness Alert", "stillness alert!!!", PRIVATE);
+        state3_stillness_timer = millis();                     // reset the stillness timer
+        max_stillness_time = &state3_max_long_stillness_time;  // set to compare against the longer stillness threshold for the rest of this session
+        stateHandler = state3_stillness;
+    }
     else if ((millis() - state2_duration_timer >= state2_max_duration) && !hasDurationAlertBeenSent) {
-        Log.warn("See duration alert, going from state3 to state2 after alert publish");
-        publishStateTransition(3, 2, checkDoor.doorStatus, checkINS.iAverage);
+        Log.warn("duration alert, remaining in state3 after publish");
+        publishStateTransition(3, 3, checkDoor.doorStatus, checkINS.iAverage);
```

Comment:
I changed this because there isn't really any reason to go back into the Duration state, we didn't see any movement. So it would have just transitioned from Duration state back to Stillness. Might as well just stay in Stillness.

Reply:
Yeah that's great! The transitions between duration <-> stillness were confusing me when I was testing my sensor earlier so I'm glad you made this change!